oidc-fapi-conformance-tests/export_results_fapi.py

```
@@ -35,14 +47,20 @@ def get_failed_tests(plan):
 
 
 if __name__ == '__main__':
+    print(f"[DEBUG] Starting export script with URL: {conformance_suite_url}")
     failed_plan_details = dict()
     contains_fails = False
     warnings.filterwarnings("ignore")
-    plan_list = json.loads(requests.get(url=conformance_suite_url + "/api/plan?length=50", verify=False).content)
+    plan_response = requests.get(url=conformance_suite_url + "/api/plan?length=50", verify=False)
```

Ruff (0.14.10)

Probable use of `requests` call without timeout

(S113)